analyzers/src/SonarAnalyzer.CSharp/Rules/LoggerMembersNamesShouldComply.cs

```diff
@@ -37,7 +37,8 @@ public sealed class LoggerMembersNamesShouldComply : ParametrizedDiagnosticAnaly
         "logger",
         "Logger",
         "_logger",
-        "_Logger");
+        "_Logger",
+        "Instance"); // "Instance" is a common name for singleton pattern
```

Comment:
What about `instance`? And while thinking about this, shouldn't we use [.WithComparer](https://learn.microsoft.com/en-us/dotnet/api/system.collections.immutable.immutablehashset-1.withcomparer?view=net-8.0#system-collections-immutable-immutablehashset-1-withcomparer(system-collections-generic-iequalitycomparer((-0)))) and provide a case insensitive compare? It will reduce the number of options and be a bit more flexible with what we accept.

Reply:
I am not sure about this. 
Would you say `_lOgGer` is a valid name?

For `instance` it makes more sense, but singletons are public, and public members almost always begin with an upper-case letter.

Reply:
> Would you say _lOgGer is a valid name?

No, I would not say that :D 

> For instance it makes more sense, but singletons are public, and public members almost always begin with an upper-case letter.

I would still add it. Different projects have different configurations. Some might use a backing field as well for the property (e.g., for cache purposes when combined with Lazy<>, ...)

Reply:
So if I understand correctly, you are suggesting to just add `instance` on the whitelist, but not use the EqualityComparer, right?